snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -68,6 +69,8 @@ def __init__(self, *, output, source, project_options,
 
         if not remote:
             remote = _get_default_remote()
+        elif remote == 'multipass':
+            remote = self._get_multipass_remote()
```

Comment:
Was this discussed before? multipass is not a container, so I imagined it would have a different way to be called. Like SNAPCRAFT_MULTIPASS=1 or something.